frontend/packages/ux-editor/src/components/config/ConditionalRenderingComponent.tsx

```diff
@@ -213,7 +213,7 @@ class ConditionalRendering extends React.Component<
     );
   };
 
-  public renderCondtionalRenderingTargetContainerOptions = (
+  public renderConditionalRenderingTargetContainerOptions = (
```

Comment:
Is it possible to test the lines that Codecov complains about in this file? 😄 
Line ```216```, ```226```, ```239```, ```248```, and ```253```.